app.py

```diff
@@ -57,6 +58,14 @@ def create_app(environment="Development"):
         '/api/v1/activity-types/', endpoint='get_activity_types'
     )
 
+    # logged activities
+    api.add_resource(
+        UserLoggedActivitiesAPI,
+        '/api/v1/users/<string:user_id>/logged-activities',
+        '/api/v1/users/<string:user_id>/logged-activities',
+        endpoint='get_user_logged_activities'
```

Comment:
I would think that under the `Logged Activities` resource there will be several endpoints utilized not just the `get` one. Isn't naming this as `get_user_logged_activities` making it specific? I would expect that something like `logged_activities` is a better option.

Reply:
Working on it.